includes/class-sensei-question.php

```diff
@@ -722,12 +741,13 @@ public static function answer_feedback_notes( $question_id ) {
 		 * Allow dynamic overriding of whether to show question answers or not
 		 *
 		 * @since 1.9.7
+		 * @hook sensei_question_show_answers
 		 *
-		 * @param boolean $show_answers
-		 * @param integer $question_id
-		 * @param integer $quiz_id
-		 * @param integer $lesson_id
-		 * @param integer $user_id
+		 * @param {bool}     $show_answers Whether to show the answer to the question.
+		 * @param {int}      $question_id  Question ID.
+		 * @param {int}      $quiz_id      Quiz ID.
+		 * @param {bool|int} $lesson_id    Lesson ID.
```

Comment:
I don't think it is possible for the `$lesson_id` to be false as we return earlier.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/3569/commits/38f28a29416247db0646b5fb02cf51ec03de38f3.